### DISCUSSION TITLE
Updated Russian translation for 0.49

### DISCUSSION THREAD

**Discussion by SanokKule** [rejetto/hfs#366] [General]
[hfs-lang-ru v2.2.zip](https://github.com/rejetto/hfs/files/12750505/hfs-lang-ru.v2.2.zip)

**Comment by rejetto** [comment]
thanks, merged

**Comment by PH5HP** [comment]
Hi,

Maybe an idea to annouce it if changes or additions to the language file being made and which ones. I would speed up the updates of the language files at least at my side. 

Kind regards,
Henk - ftp.ph5hp.nl
